common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3325,6 +3326,15 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val TEST_WORKER_UNDER_TEST: ConfigEntry[Boolean] =
```

Comment:
why this configuration needed?

Reply:
Some UTs will fail without this config, they don't really call commitFiles.